host_cli/data_core_wrapper.cpp

```diff
@@ -66,4 +67,26 @@ absl::Status DataCoreWrapper::WriteNewGfxrFile(const std::string& new_gfxr_file_
     return absl::OkStatus();
 }
 
+absl::Status DataCoreWrapper::RemoveGfxrBlocks(std::vector<int> block_ids)
+{
+    assert(m_data_core != nullptr);
+    if (!IsGfxrLoaded())
+    {
+        return absl::FailedPreconditionError("Must load original GFXR first");
+    }
+
+    std::shared_ptr<gfxrecon::decode::DiveBlockData>
+    dive_block_data = m_data_core->GetMutableCaptureData().GetMutableGfxrData();
+
+    for (const auto& id : block_ids)
+    {
+        bool res = dive_block_data->AddModification(id, 0, nullptr);
+        if (!res)
+        {
+            return absl::InternalError("Could not delete block id: " + id);
```

Comment:
```
[593/761] Building CXX object host_cli/CMakeFiles/data_core_wrapper_lib.dir/data_core_wrapper.cpp.o
/home/runner/work/dive/dive/host_cli/data_core_wrapper.cpp:86:70: warning: adding 'int const' to a string does not append to the string [-Wstring-plus-int]
            return absl::InternalError("Could not delete block id: " + id);
                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~
/home/runner/work/dive/dive/host_cli/data_core_wrapper.cpp:86:70: note: use array indexing to silence this warning
            return absl::InternalError("Could not delete block id: " + id);
                                                                     ^
                                       &                             [   ]
```
Consider absl::StrCat to convert the int into a string:
```suggestion
            return absl::InternalError(absl::StrCat("Could not delete block id: ", id));
```